### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JosephBJoyce**
Fixed some 'copy-paste' errors in the import-postman-collection section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,15 +153,15 @@ From SwaggerHub Explore, navigate to your browser development tools, locate the
                 |_|
 ```
 **Description:**
-  > Import SwaggerHub Explore spaces from a file
+  > Import Postman collections from a file
```
Comment (frankkilcommins) [inline README.md:156]:
```suggestion
  > Import Postman collections (v2.1) from a file
```

Comment (frankkilcommins) [reply README.md:156]: Thanks @JosephBJoyce 

One minor suggestion above, and also could you update the text description in line 29 to state: "Import a Postman collection v2.1 into SwaggerHub Explore"

Comment (JosephBJoyce) [reply README.md]: Done and done!

**Review by frankkilcommins — APPROVED** [review]
LGTM! 

This will be included in the next release to update the Nuget documentation. Let me know if you hear of any user complaints and I can push a release sooner.

**Pull request merged**